src/interfaces/records/messages/records-read.ts

```diff
@@ -55,12 +55,9 @@ export class RecordsRead {
       date      : options.date ?? currentTime
     };
 
-    const message: RecordsReadMessage = { descriptor };
-
-    if (options.authorizationSignatureInput) {
-      const authorization = await Message.signAsAuthorization(descriptor, options.authorizationSignatureInput);
-      message.authorization = authorization;
-    }
+    // FIXME: what if authorizationSignatureInput is undefined?
```

Comment:
We should make `authorizationSignatureInput` required in `RecordsQueryOptions`. Every other message has it required, so it's probably a typo that it's optional here.

Reply:
Done.